tools/ratbagctl.in.in

```diff
@@ -229,6 +229,7 @@ def print_device(d, level):
 
     print(" " * level + "{} - {}".format(d.id, d.name))
     print(" " * level + "             Model: {}".format(d.model))
+    print(" " * level + "       Device Type: {}".format(d.device_type.name))
```

Comment:
TODO: find a prettier way to format this, because right now this shouts "MOUSE".

Reply:
Python's `str.capitalize()` will make this less agressive :)

Reply:
Thanks!